index.html

```diff
@@ -396,7 +424,132 @@
+                                            ]
+                             },
+                properties : "string", 	
+                visible    : true,	
+                highlight  : true
+        };
+                                
+        itowns.viewer.addFeature(feature); 
+        
+        //Feature layers used fot tests purpose
+        itowns.viewer.addFeatureLayer({
+            id          : 'testWfsPoint',
+            protocol    : "wfs",
+            url         : 'https://download.data.grandlyon.com/wfs/rdata?',
+            title       : 'jcd_jcdecaux.jcdvelov',
+            crs         : 'EPSG:4326',
+            type        : 'point',
+            version   : '2.0.0',
+            options: {
+                mimetype  : "geojson",
+                tileMatrixSet: 'WGS84G'
+            },
+            params: {
+                color: {
+                    property: 'availability',
+                    testTab : ['Vert', 'Orange', 'Bleu', 'Gris'],
+                    colorTab: [0x339933, 0xFF9900, 0x3399FF, 0x888888]
+                },
+                heightParam : 'pop_rank',
+                nbSegment   : 8,
+                level: 14,
+                // Test of retail change on a specified area (box area define by 2 angles : min and max)
```

Comment:
Jeremy told me to do another branch with the change details. I will tell you when I am done with it.